src/taipy/rest/api/resources/cycle.py

```diff
@@ -104,8 +110,11 @@ class CycleList(Resource):
     get:
       tags:
         - api
-      summary: Get a list of cycles
-      description: Get a list of paginated cycles
+      summary: Get all cycles
+      description: |
+        Return all cycles.
+
+        In the Enterprise version, this endpoint requires TAIPY_READER role.
```

Comment:
"In the Enterprise version, this endpoint requires TAIPY_READER role"
->
"In the Enterprise edition, this endpoint requires the *TAIPY_READER* role"

(and everywhere in all files)